tap_xero/client.py

```diff
@@ -255,7 +258,8 @@ def filter(self, tap_stream_id, since=None, **params):
             headers["User-Agent"] = self.user_agent
         if since:
             headers["If-Modified-Since"] = since
-
+        if tap_stream_id == "journals_payments_only":
+            params.update({"paymentsOnly":True})
```

Comment:
Considering this is case-sensitive, we tried with only `true`
should this be updated to that as well?

Reply:
Hi @irfu-dt ideally It shouldn't matter but for sake of replicating testing params. I have revised my MR.